internal/configuration/settings/portforward.go

```diff
@@ -29,6 +29,11 @@ type PortForwarding struct {
 	// to write to a file. It cannot be nil for the
 	// internal state
 	Filepath *string `json:"status_file_path"`
+	// Command is the port forwarding status command
+	// to use. It can be the empty string to indicate not
+	// to run a command. It cannot be nil for the
+	// internal state
+	Command *string `json:"status_command"`
```

Comment:
Sorry to re-iterate on this, can you rename this to `UpCommand`? To plan ahead for an eventual `DownCommand` 🤔 Thanks!